### PR TITLE
feat: Enhance interactive queries; Suppress non-data output

### PR DESCRIPTION
**Pull request by yuryfirebolt**
The most major enhancement is seen in the screenshot:
- Print the execution result of the query (Error/Success).
- Print output information only from queries, when data is expected.
- In case of multiple queries, print the result of queries until we encounter error. 

Screenshot 2022-04-27 at 10 55 17

### CODE REVIEW

File: tests/unit/test_query.py [hunk 1]
```diff
@@ -244,19 +244,21 @@ def test_sql_execution_multiline(
...
     assert result.exit_code == 0
 
     assert cursor_mock.nextset.call_count == 2
     assert cursor_mock.fetchall.call_count == 2
 
-    cursor_mock.execute.assert_called_once_with(expected_sql)
+    cursor_mock.execute.assert_has_calls(
+        [mock.call("SELECT * FROM t1;"), mock.call("SELECT * FROM t2;")]
+    )
+    assert cursor_mock.execute.call_count == 2
```
Comment (ptiurin) [inline tests/unit/test_query.py:261]: You can combine those two by using `mock_calls` e.g.
```
assert cursor_mock.execute.mock_calls == [mock.call("SELECT * FROM t1;"), mock.call("SELECT * FROM t2;")]
```

File: tests/unit/test_utils.py [hunk 2]
```diff
@@ -426,3 +427,17 @@ def test_create_aws_creds_from_environ_invalide():
...
+    test common cases of format_short_statement
+    """
+    assert format_short_statement("SELECT 1") == "SELECT 1"
+    assert format_short_statement("/**/ SELECT 1") == "SELECT 1"
+    assert format_short_statement("-- SELECT 1;\nSELECT 2") == "SELECT 2"
+    assert (
+        format_short_statement("SELECT        23          \n FROM table\n")
+        == "SELECT 23 FROM table"
+    )
+    assert format_short_statement("SELECT 123", truncate_long_string=6) == "SELECT ..."
```
Comment (ptiurin) [inline tests/unit/test_utils.py:443]: Matter of style, but I prefer using `pytest.mark.parametrize` when there's repetitive input/output checking.

File: src/firebolt_cli/utils.py [hunk 3]
```diff
@@ -46,6 +48,32 @@ def get_command(self, ctx: Context, cmd_name: str) -> Optional[Command]:
...
+
+    Returns: a formatted string
+
+    """
+    statement = sqlparse.format(
+        str(statement), strip_comments=True, use_space_around_operators=True
+    )
+
+    statement = statement.replace("\n", " ").replace("\t", " ").strip()
+    statement = re.sub(" +", " ", statement)
```
Comment (ptiurin) [inline src/firebolt_cli/utils.py:69]: Why are we stripping `+`? Am I missing something?

File: src/firebolt_cli/query.py [hunk 4]
```diff
@@ -33,28 +35,61 @@
...
+    print execution summary result: Success or Error
+    and a shortened statement to which it is referring
+    """
+    statement = format_short_statement(statement)
+    msg, color = ("Success:", "green") if success else ("Error:", "yellow")
+
+    echo("{} {}".format(click.style(msg, fg=color, bold=True), statement))
+
+
+def execute_and_print(cursor: Cursor, query: str, use_csv: bool) -> None:
```
Comment (ptiurin) [inline src/firebolt_cli/query.py:57]: We should expand the test coverage on this function. e.g. when Exception is raised and some of the if conditions.

**Comment by ptiurin** [comment]
Why is data outputted twice here?
image

File: src/firebolt_cli/utils.py (hunk 3 above)
Comment (yuryfirebolt) [reply src/firebolt_cli/utils.py:69]: we are stripping consecutive whitespaces, I will add comment to make it more explicit

File: tests/unit/test_utils.py (hunk 2 above)
Comment (yuryfirebolt) [reply tests/unit/test_utils.py:443]: Will change it

File: src/firebolt_cli/query.py (hunk 4 above)
Comment (yuryfirebolt) [reply src/firebolt_cli/query.py:57]: The coverage of this function is 100% via query and interactive query tests

**Comment by yuryfirebolt** [comment]
> 

The data column name automatically gets the value of the first row.

File: src/firebolt_cli/utils.py (hunk 3 above)
Comment (ptiurin) [reply src/firebolt_cli/utils.py:71]: Silly me, it's a regex, duuh.

**Review by ptiurin — APPROVED** [review]

**Pull request merged**